test/integration/features/cmd-config.feature

```diff
@@ -203,6 +203,7 @@ user defined options which changes default behaviour of Minishift.
   Scenario: Minishift informs about starting with correct setup of memory, disk and CPU
   Note: Minishift rounds the values for the report to make it more readable.
         However original non-rounded values are used for the startup.
+    Given image caching is disabled
```

Comment:
This will prevent the default configuration to run if this feature is only executed i,e make integration GODOG_OPTS="-tags cmd-config". 
Same is applicable for all feature file

Reply:
Yes, a side effect of this current implementation. Can't do much here I guess.

Reply:
Actually its blocking a single feature file run with default configuration. This should be handle somehow smartly otherwise single feature run will be executed without image caching enable (Here basic can run with default image caching enabled). 

We should wait for more feedback on this, else will document it or open a new issue to refactor it later 
@budhrg WDYT ?

Reply:
AFAIU this strategy is similar to what we did with proxy. It could be run with all features, but to be clear, we made it into just one feature, which focuses on it. So it is expected that `cmd-config.feature` to not run with image caching. However it is true, that in basic.feature we do not test that much - for example a second startup on which we would check that the caching actually happens is missing. This might be fixed in follow up issue in `cmd-image.feature`.